app/views/sessions/new_sign_up.html.erb

```diff
@@ -0,0 +1,11 @@
+.
+<h2>SIGN UP</h2>
+
+
+<%= simple_form_for @user do |f| %>
+  <%= f.input :email %>
+  <%= f.input :password %>
+  <%= f.input :password_confirmation %>
```

Comment:
add validation on frontend side for checking if password and password_confirmation are same